docs/usage/logging.md

```diff
@@ -0,0 +1,126 @@
+# Logging stack
+
+### When to use the logging stack?
```

Comment:
Sounds like there are cases when I might not want to use it?
Consider to not use a dedicated heading (i.e. under the title) or something like:
```suggestion
### Motivation
```
```suggestion
### Why logging stack?
```
```suggestion
### Introduction
```
```suggestion
### Overview
```